### PR TITLE
faster callcenter queries using new denrom columns

### PR DESCRIPTION
**Pull request by snopoke**
on hold until I've backfilled the new columns added here: https://github.com/dimagi/commcare-hq/pull/6885

### CODE REVIEW

File: corehq/apps/callcenter/indicator_sets.py [hunk 1]
```diff
@@ -381,11 +374,16 @@ def add_cases_active_data(self, range_name, lower, upper):
...
+            .exclude(case_type=self.cc_case_type) \
             .filter(
-                actions__date__gte=lower,
-                actions__date__lt=upper
-            ).annotate(count=Count('case_id', distinct=True))
+                domain=self.domain,
+                case_owner__in=self.owners_needing_data,
+                date__gte=lower,
+                date__lt=upper
+            ).annotate(count=Count('case', distinct=True))
```
Comment (czue) [inline corehq/apps/callcenter/indicator_sets.py]: (not blocking) does the ordering of the different changes matter here? Just wondering whether it's useful to reuse any of the shared query params similar to how they shared code previously, though not obviously a better change to me.

**Comment by czue** [comment]
:+1: to merge whenever the data is ready. assume the tests are sufficient for changes?

File: corehq/apps/callcenter/indicator_sets.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/callcenter/indicator_sets.py]: Not really worth trying to share the params I don't think. It also makes it harder to figure out what the query is.

**Comment by snopoke** [comment]
Thanks, still backfilling data. Yes, I think the tests are sufficient.

**Comment by TylerSheffels** [comment]
Looks good to me

**Comment by snopoke** [comment]
@czue @TylerSheffels this is GTG

**Pull request merged**